answers.txt

```diff
@@ -0,0 +1,31 @@
+<<<<<<< HEAD
+1.Gitignore files displayed by Git are neglected untracked files.These files consist of patterns from various sources in the higher level to lower level order.The patterns in these files are used to decide a path.These files do not affect the tracked files on Git.
+A gitignore file for Java project includes the following:
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+2.When a branch with one or more commit is ahead of the master branch at the time of merging ,Git performs a process called fast forward to match the merged commits.
+
+3.For development process ,there has to be different branches in a Git repository.Whenever there is a need to change the code or fix a bug,a new branch is created in order to make changes.After reviewing the changes ,the branch can be merged with the present master branch.Every new branch represents pointers to a new commit.
```

Comment:
Branches help multiple people to work in parallel. And the maintenance of code becomes easier.